app/controllers/report_controller/pivot_options.rb

```diff
@@ -1,4 +1,6 @@
 class ReportController
+  include ReportHelper
```

Comment:
This looks like you're including `ReportHelper` for the whole `ReportController` anyway .. move it to `report_controller.rb` please...